tools/concat/.shed.yml

```diff
@@ -0,0 +1,7 @@
+categories: [Assembly]
+description: concatenate
+name: concat
```

Comment:
change name to **concat_paired** and change tool directory name as well to match